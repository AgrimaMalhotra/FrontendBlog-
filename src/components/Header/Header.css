*{
    margin: 0px;
} 

.header-section{
    background-color: black;
    color: white;
    font-family: 'Tahoma', sans-serif;
    padding: 25px 220px 5px 220px;
}

.header-section > .heading{
    font-size: 25px;
    font-weight: normal;
}

.header-section > .sub-heading{
    font-size: 18px;
    font-style: italic;
    font-weight: lighter;
}

.nav-bar{
    display: flex;
    justify-content: flex-end;
}

.nav-bar > a{
    text-decoration: none;
    color: white;
    padding-left: 15vw;
}
@media (max-width: 480px) {
    .header-section {
        font-size: 3.2rem;
    }
    .nav-bar{
        font-size: 3.2rem;
    }
}