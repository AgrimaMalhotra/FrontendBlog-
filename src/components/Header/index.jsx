import * as React from 'react';
import './Header.css';

function Header() {
  return (
    <div className="header-section basic-padding">
      <p className="heading">The Artifact</p>
      <p className="sub-heading">Culture & Art blog</p>
      <nav className="nav-bar">
        <a href="#Blog"> Blog</a>
        <a href="#About">About</a>
        <a href="#Contact">Contact</a>
      </nav>
    </div>
  );
}

export default Header;
