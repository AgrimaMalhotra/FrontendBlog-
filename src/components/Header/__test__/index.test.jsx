import { render } from '@testing-library/react';
import * as React from 'react';
import Header from '..';

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
