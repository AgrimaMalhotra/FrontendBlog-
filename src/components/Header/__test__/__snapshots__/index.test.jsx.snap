// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header-section"
  >
    <p
      class="heading"
    >
      The Artifact
    </p>
    <p
      class="sub-heading"
    >
      Culture & Art blog
    </p>
    <nav
      class="nav-bar"
    >
      <a
        href=""
      >
         Blog
      </a>
      <a
        href=""
      >
        About
      </a>
      <a
        href=""
      >
        Contact
      </a>
    </nav>
  </div>
</DocumentFragment>
`;
